Presentation/MainPDF/ViewModel/BackPageBtnViewModel.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  BackPageButtonViewModel.swift
+//  Reazy
+//
+//  Created by 김예림 on 1/31/25.
+//
+
+import Foundation
+import PDFKit
+
+@MainActor
+class BackPageBtnViewModel: ObservableObject {
+    private let BackPageBtnUsecase: BackPageBtnUseCase
+    
+    init(BackPageBtnUsecase: BackPageBtnUseCase) {
+        self.BackPageBtnUsecase = BackPageBtnUsecase
+    }
+    
+    @Published var backPageDestination: PDFDestination?
+    @Published var backScaleFactor: CGFloat = .zero
+    @Published var isLinkTapped: Bool = false
+    
+    func updateTempDestination(_ destination: PDFDestination) {
+        BackPageBtnUsecase.updateTempDestination(destination)
+    }
+    
+    func updateBackDestination() {
+        if let destination = BackPageBtnUsecase.tempDestination {
+            backPageDestination = BackPageBtnUsecase.convertDestination(for: destination)
+        }
+    }
+    
+    func setDestination(pdfView: PDFView) {
+        if let destination = BackPageBtnUsecase.getTopLeadingDestination(pdfView: pdfView, scaleFactor: backScaleFactor) {
+            BackPageBtnUsecase.updateTempDestination(destination)
+        }
+    }
+    
+    func handleButnVisible() {
```

Comment:
해당 메소드만 Btn이 아닌 Butn으로 하셨는데 통일 하는 것이 좋아보입니다!!

```suggestion
    func handleBtnVisible() {
```